docs/intro/tutorial07.txt

```diff
@@ -418,5 +418,44 @@ uses a template variable called ``app_list``. That variable contains every
 installed Django app. Instead of using that, you can hard-code links to
 object-specific admin pages in whatever way you think is best.
 
+How to completely delete an app from the project
+================================================
+Now that you have learned every aspect of app creation, customization and etc.
+You need to know how to properly delete an app from the project.
+
+First, you need to remove all the tables from your database or any other traces 
+the app might have. For Django version older than 1.7, use:
+.. console::
```

Comment:
Django < 3.1 is not supported anymore, we don't include snippets for unsupported versions.